waffle-compiler/src/compileDockerVyper.ts

```diff
@@ -0,0 +1,27 @@
+import {Config} from './config';
+import {execSync} from 'child_process';
+import {ImportFile} from '@resolver-engine/imports';
+import {solcOutputMaxBuffer} from './compiler';
+import {buildInputObject} from './buildUitls';
+
+const CONTAINER_PATH = '/project';
+
+export function compileDockerVyper(config: Config) {
+  return async function compile(sources: ImportFile[]) {
+    const command = createBuildCommand(config);
```

Comment:
add warning that this is experimental and the api might change without major version change.